keepassxc-browser/content/totp-field.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const ignoreRegex = /(zip|postal).*code/i;
+const ignoreRegex = /(act|bank|coupon|postal|user|zip).*code|comment|author/i;
```

Comment:
How is that regex evaluated? My read is that it would match "zipcomment" and "bankauthor" but I don't think that's the intention.

Reply:
Related, if we are going to match a name we should make it an inclusion instead of exclusion. My thought is there are far more false positives on "****code****" then true positives like "pincode" or "totpcode"

Reply:
How about setting up a form/issue/wikipage/json_file for users to report this kind of information for the sites they care about? That should then serve as the basis for regression tests.

For that matter, protonmail uses "twoFactorCode".

Reply:
We are doing both inclusion and exclusion. I'd like to keep it that way. The Regex must be added to the test PR later, but must be tested properly for this PR first.

Reply:
ok no problems thanks!